_posts/unixEnthusiast/2022-09-22-first-post-unixEnthusiast.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+author: <unixEnthusiast>
```

Comment:
The angle brackets aren't needed here, just the name. I'll make this small change and merge!